exercises/Perception-Driven_Manipulation/template_ws/src/robot_config/urdf/ur5.urdf.xacro

```diff
@@ -1,70 +1,9 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
Was receiving the error: Undefined parameters [hw_interface] when instantiating macro: ur_arm_transmission.

To resolve the error, substituted the existing ur5.urdf.xacro with `universal_robot/ur_description/urdf/ur5.urdf/xacro`